trfind/finders/stephabegg.py

```diff
@@ -10,14 +10,15 @@
 from urllib.parse import urljoin, urlencode
 from urllib.request import Request, urlopen
 
-from trfind.models import TripReportSummary
+from ..models import TripReportSummary
 
 STEPH_ABEGG_SITE = 'Steph Abegg'
 
 def remove_mount(title_lowered):
     ''' Page titles incldue stuff like 'mt. stuart' so when we search in the title, remove 'Mount' from our peak name '''
     if title_lowered.startswith('mount'):
         return title_lowered[len('mount'):]
+    return title_lowered
```

Comment:
oops! nice catch. maybe add a regression test?

Reply:
added a test to test_shared.py and it passes